test/javascript.js

```diff
@@ -85,6 +85,58 @@ describe('javascript', function() {
     );
   });
 
+  it('should handle duplicate plugins from preset env for babel v6', async function() {
```

Comment:
This test seems to pass, even without the code changes. Are you sure this PR is needed?

Reply:
This PR or this test ?

I know this test is passing without this PR (but not another one I've also added), but why we should exclude it, since it covers codebase for one more scenario ?

Reply:
second test `should handle duplicate plugins from preset env for babel v7` will not pass without this PR

Reply:
But I found this test is broken actually, also as getting corresponding babel 7 plugin name from babel 6. I'll fix it

Reply:
@devongovett fixed. Also I moved tests for `transforms/babel/config.js` to own test file `getBabelConfig.js`

Reply:
and tests are more accurate now